modules/@apostrophecms/admin-bar/ui/apos/components/TheAposAdminBar.vue

```diff
@@ -136,6 +138,22 @@ export default {
       return this.moduleOptions.contextEditorName;
     }
   },
+  watch: {
+    async idleTriggered(newVal, oldVal) {
+      if (newVal && !oldVal) {
+        const response = await apos.confirm({
+          heading: 'You have been idle for 30 minutes with unsaved changes. Please save to avoid losing your updates.',
```

Comment:
How about saving for them on OK? If I saw this prompt and clicked OK I might assume it saved for me.

Reply:
... Which would make it more of a true confirm prompt.